contrib/otlp/Exporter.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\otlp;
```

Comment:
is it typo or really lowercase `otlp` ?

Reply:
lowercase

Reply:
Other languages also use lower case, but I'm wondering if it's more PHP like to use capitals here.  I don't have strong opinions.

Reply:
imo StuldyCase is preferred more than camelCase or lowercase in class name and inamespace.

Reply:
StudlyCase would be fine with me too:

`namespace OpenTelemetry\Contrib\Otlp;`

Reply:
updated the namespace have a look and thanks for this :+1: